# Define the goal
bool command
float64 x_input
float64 y_input
float64 z_input
float64 roll_input
float64 pitch_input
float64 yaw_input
float64 check
float64 sim_vel
string constraint
float64[] constraints_values
float64[] enable_joints
float64[] orientation_constraints_values
float64 position_constraint_value
---
# Define the result
float64[] result
---
# Define a feedback message
string feedback
